### ISSUE TITLE
post-IF with implicit function call fails

### ISSUE THREAD

**Issue by eventualbuddha** [decaffeinate/decaffeinate#327]
decaffeinate is crashing on my CoffeeScript input:

```coffee
a if b c
```

[(repl)](http://decaffeinate-project.org/repl/#?evaluate=true&code=a%20if%20b%20c%0A)

I get this error:

```
unexpected ')' at 14
```

This is because when replacing the whole post-IF the extra `)` doesn't come along for the ride:

```
NormalizeStage a if b c

FunctionApplicationPatcher OVERWRITE [6, 7) " " → "("
FunctionApplicationPatcher INSERT RIGHT 8 ")" BEFORE "\n"
ConditionalPatcher OVERWRITE [0, 8) "a if b c" → "if b(c) then a"
MainStage if b(c) then a)
```

**Comment by eventualbuddha** [comment]
As a workaround, manually re-writing `a if b` to `if b then a` first will make this work.

**Comment by eventualbuddha** [comment]
Fixed in v2.17.4.
